i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,60 @@ sidebar_position: 4
 
 # Naming
 
-## Problem
+Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities [are called differently][disc-src]. For example:
 
-Due to the fact that each developer has his own experience and development context - [we are used to calling the same entities differently][disc-src], which can lead to misunderstandings within the team.
+- Components for display can be called "ui", "components", "ui-kit", "views", ...
+- The code that is reused throughout the application can be called "core", "shared", "app", ...
+- Business logic code can be called "store", "model", "state", ...
 
-- *Components to be displayed - `ui` / `components` / `ui-kit` / `views` / ...*
-- *Code reusable in all parts of the application - the `core`/ `shared` / `app` / ...*
-- *Code business logic `store` / `model` / `state` / ...*
+## Naming in Feature-Sliced Design {#naming-in-fsd}
 
-## Naming in FSD
+The methodology uses specific terms such as:
 
-The methodology uses such terms as
+- [Layers][refs-layers] — "app", "process", "page", "feature", "entity", "shared"
+- [Slices][refs-slices]
+- [Segments][refs-segments] — "ui', "model", "lib", "api", "config"
 
-- `app`, `process`, `page`, `feature`, `entity`, `shared` - *[layers][refs-layers]*
-- `ui`, `model`, `lib`, `api` - *[segments][refs-segments]*
+These terms are [clearly defined][refs-reference] in the methodology.
 
-Within the framework of the methodology, each of these terms has a [clear definition][refs-reference]
+When working on a project based on the **Feature-Sliced Design** methodology, it is important to use [original naming][disc-src] to avoid confusion among team members and others. This is especially important when a new developer who is familiar with the methodology joins the project, or when he asks for help from the community.
```

Comment:
suggestion (non-blocking): remove a link and slightly reduce the length
```suggestion
When working on a project based on the **Feature-Sliced Design** methodology, it is important to use original naming to benefit from the reduction of project knowledge and to make it easier to ask for help from the community.
```

Reply:
I rewrote this piece a little bit